holoviews/plotting/mpl/util.py

```diff
@@ -334,3 +337,36 @@ def polygons_to_path_patches(element):
             subpath.append(PathPatch(Path(vertices, codes)))
         mpl_paths.append(subpath)
     return mpl_paths
+
+try:
+    import cftime
+    from nc_time_axis import NetCDFTimeConverter, CalendarDateTime
+    nc_axis_available = True
+except:
+    from matplotlib.dates import DateConverter
+    NetCDFTimeConverter = DateConverter
+    nc_axis_available = False
+
+
+class CFTimeConverter(NetCDFTimeConverter):
+    """
+    Defines conversions for cftime types by extending nc_time_axis.
+    """
+
+    @classmethod
+    def convert(cls, value, unit, axis):
+        if not nc_axis_available:
+            raise ValueError('In order to display cftime types with '
+                             'matplotlib install the nc_time_axis '
+                             'library using pip or from conda-forge '
+                             'using:\n\tconda install -c conda-forge '
+                             'nc_time_axis')
+        if isinstance(value, cftime_types):
+            value = CalendarDateTime(cftime.datetime(*value.timetuple()[0:6]), value.calendar)
```

Comment:
Is there a reason for converting to the generic `cftime.datetime` here and below?  I think you can just pass `value` and `value.calendar` to the `CalendarDateTime` constructor.

Reply:
Thanks, don't think there was any good reason.